src/Details.js

```diff
@@ -1,8 +1,25 @@
 import React from 'react'
+class app extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+        message: `Hello, this will be the details page 
+        for each Movie & TV show :-)`
+        }
+    }
+componentDidMount(){ //waits for the componetent to mouted(initiated) and runs the code inside
```

Comment:
Please add a space before the bracket. I suggest to format the document automatically by `right click` -> `Format document`